service/history/stateBuilder.go

```diff
@@ -26,17 +26,19 @@ import (
 	"time"
 
 	"github.com/pborman/uuid"
+
```

Comment:
nit: remove empty line

Reply:
this is due to the new goimports sorting